guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -8,7 +8,7 @@ ifdef::satellite[]
 The operating system image is based on {RHEL} 7.
 endif::[]
 ifndef::satellite[]
-The operating system image is based on Cent OS.
+The operating system image is based on CentOS.
```

Comment:
Maybe even say "CentOS 8"? cc @lzap 

source: https://community.theforeman.org/t/announcing-foreman-discovery-image-4-0/26817

Reply:
I like that, but then we should probably explicitly mention that it's stream.

Reply:
Discovery is CentOS 8 Stream upstream, RHEL 7 downstream (RHEL 8 in the works).

Reply:
Thanks for confirming, will update.